baseline_stability.py

```diff
@@ -14,55 +13,61 @@
-    # parser.add_option("-o", "--output", dest="output", help="output filename", default="output_crab.txt", type=str)
-    # parser.add_option("-d", "--display", dest="display", action="store_true", help="Display rather than output data",
-    #                   default=False)
-    parser.add_option('-s', "--file_start", dest='file_start', help='file number start', default=0, type=int)
-    parser.add_option('-e', "--file_end", dest='file_end', help='file number end', default=0, type=int)
-    parser.add_option('-c', "--camera_config", dest='camera_config_file', help='camera config file to load Camera()'
-                      , default='digicampipe/tests/resources/camera_config.cfg')
+    parser.add_option(
+        "-p", "--path", dest="directory", help="directory to data files",
+        default=os.path.realpath('digicampipe/tests/resources'))
+    parser.add_option(
+        '-f', '--file', dest='filename',
+        help='file basename e.g. CRAB_01_0_000.%03d.fits.fz',
+        default='/example_100_evts.%03d.fits.fz', type=str)
+    parser.add_option(
+        '-s', "--file_start", dest='file_start',
+        help='file number start', default=0, type=int)
+    parser.add_option(
+        '-e', "--file_end", dest='file_end',
+        help='file number end', default=0, type=int)
 
     (options, args) = parser.parse_args()
 
     # Input/Output files
     directory = options.directory
     filename = directory + options.filename
-    file_list = [filename % number for number in range(options.file_start, options.file_end + 1)]
-    digicam_config_file = options.camera_config_file
-
-    # Camera and Geometry objects (mapping, pixel, patch + x,y coordinates pixels)
-    digicam = Camera(_config_file=digicam_config_file)
-    digicam_geometry = geometry.generate_geometry_from_camera(camera=digicam)
+    file_list = [
+        filename % number
+        for number in range(options.file_start, options.file_end + 1)
+    ]
 
     # Noisy pixels not taken into account
-    # pixel_not_wanted = [1038, 1039, 1002, 1003, 1004, 966, 967, 968, 930, 931, 932, 896]
+    # pixel_not_wanted = [
+    #    1038, 1039, 1002, 1003, 1004, 966, 967, 968, 930, 931, 932, 896]
     n_events = zfits.count_number_events(file_list=file_list)
     ####################
     ##### ANALYSIS #####
     ####################
-    digicam_baseline = np.zeros((len(digicam_geometry.pix_id), n_events))
+    baseline = np.zeros((len(DigiCam.geometry.pix_id), n_events))
 
     # Define the event stream
-    data_stream = event_stream(file_list=file_list, expert_mode=True, camera_geometry=digicam_geometry, camera=digicam)
+    data_stream = event_stream(
+        file_list=file_list,
+        camera=DigiCam
+    )
 
     for i, data in zip(range(n_events), data_stream):
-
         for tel_id in data.r0.tels_with_data:
-
-            digicam_baseline[..., i] = data.r0.tel[tel_id].digicam_baseline
+            baseline[..., i] = data.r0.tel[tel_id].baseline
```

Comment:
I think here it should access: `data.r0.tel[tel_id].digicam_baseline` meaning it is the baseline computed by DigiCam FPGA.
or it was renamed...

Reply:
oh .. yeah that was a mistake! Good catch!

I thought the variable name is too long and I can make it shorter, since in the whole script there is only one "digigcam_baseline", so I thought there is not need for the extra qualification "digicam" ... since we all know we are working on digicam here... 

anyway .. I fucked that up .. damn

Reply:
Is it solved ?

Reply:
yes it is solved, but I closed this PR. Since it way too long. I replaced it with two other PRs, which are hopefully smaller and easier to review.

Reply:
It was solved directly after you commented on it in this commit:
https://github.com/cta-sst-1m/digicampipe/pull/114/commits/f470706bce30ff7b1ced95b84a1380727e1e3dd1